tests/formatter_tests.js

```diff
@@ -79,6 +83,200 @@ VF.Test.Formatter = (function() {
       equal(tickables1[2].getX(), tickables2[2].getX(), 'Last notes of both voices have the same X');
       ok(tickables1[1].getX() < tickables2[1].getX(), 'Second note of voice 2 is to the right of the second note of voice 1');
     },
+    formatAccidentalSpaces: function(options) {
```

Comment:
Maybe we should move this to `accidentals_test.js`?

Reply:
sure